docs/src/large_scale_condensation.md

```diff
@@ -127,7 +127,7 @@ The tendencies ``\delta q`` in units of  kg/kg/s are vertically
 integrated to diagnose the large-scale precipitation ``P`` in units of meters
 
 ```math
-P = -\int \frac{\Delta t}{g \rho} \delta q dp
+P = - \int_{top}^{bottom} \frac{\Delta t}{g \rho} \delta q dp 
```

Comment:
I think this is bottom to top if written down mathematically. $\delta q$ is negative in the occurence of precipitation, $dp$ is positive when integrating top to bottom (i.e. $p=0 to p=p_0$) and the fraction is positive too. So it's only the negative sign of $\delta q$ that needs to be switched to have a positive amount of precipitation accordingly.